docs/en/status-communities/change-a-token-gated-community-to-open.md

```diff
@@ -0,0 +1,43 @@
+---
+id: 637
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Change a token-gated community to open
+
+!!! note ""
+    Currently, you can only set up token-gated and open communities using Status desktop. [:octicons-desktop-download-16: Download Status desktop][status-web-download].
+
+In a [token-gated community][token-based-access-to-communities-and-channels] members should meet specific token requirements to join. As a community owner, you can change an open-gated community into an open one so anyone can join without requiring tokens.
+
+!!! note
+    In addition to token requirements, you can set up [manual approval][about-community-request-approvals] to join your community. Token requirements and manual approval work independently.
+
+You might want to transition from a token-gated community to an open one for various reasons. For instance, you might want to grow your community by attracting new members, or perhaps you want to make your community more inclusive by removing the entry barrier.
+
+!!! tip
+    You can use token-gated communities and channels separately. For example, you may want to set up an open Status Community but [restrict access to certain channels][set-up-a-token-gated-channel].
+
+## What to expect
+
+- Remove all the `Become member` community permissions to transform your token-gated community into an open one.
+- You can change your community from token-gated to open or the other way around at any time.
+- Any token-based channel within your community remains token-based unless you [change it to open][change-a-token-gated-channel-to-open].
```

Comment:
Let's say a community requires all joiners to own 10 SNT. I do own 10 SNT, so I apply to join and get accepted. All the channels are token-gated as well, but since I satisfy the requirements, I leave a few messages here and there. I then go on to sell 5 SNT. 

1. Do I get auto-kicked out of the community?
2. Can I still access the token-gated channels?

We might want to explain this in just a few sentences here or somewhere else in the article.

Reply:
_Uhm_. 🤔 I think we're mixing different things here. This is about how to open your community, not how token requirements work. But I know what you mean. I included an additional 'What to expect' with a link where these questions are discussed.